### PR TITLE
2691 DB based print files

### PR DESCRIPTION
**Pull request by AdamHawtin**
# Motivation and Context
WIP

# What has changed
- Regenerate DDL (add print file row table)

# How to test?

WIP

# Links
https://trello.com/c/Lni8psPO/2691-productionise-database-based-print-file-building-spike-8

### CODE REVIEW

File: groundzero_ddl/casev3.sql [hunk 1]
```diff
@@ -299,6 +310,11 @@ create index cases_case_ref_idx on cases (case_ref);
        foreign key (job_id) 
        references job;
 
+    alter table if exists print_file_row 
+       add constraint FKfo2cs8srx9nsfeliw3ekveehu 
+       foreign key (action_rule_id)
```
Comment (NickGrantONS) [inline groundzero_ddl/casev3.sql:315]: foreign key on action rule not required

File: groundzero_ddl/casev3.sql [hunk 2]
```diff
@@ -137,6 +137,17 @@
...
     );
 
+    create table print_file_row (
+       id  bigserial not null,
+        batch_id uuid,
+        batch_quantity int4 not null,
+        pack_code varchar(255) not null,
+        print_supplier varchar(255) not null,
+        row varchar(255),
+        action_rule_id uuid,
```
Comment (NickGrantONS) [inline groundzero_ddl/casev3.sql:147]: relationship with action rule not required

File: patches/100_add_print_file_row_table.sql [hunk 3]
```diff
@@ -0,0 +1,24 @@
+-- ****************************************************************************
+-- RM SQL DATABASE UPDATE SCRIPT
+-- ****************************************************************************
+-- Number: 100
```
Comment (NickGrantONS) [inline patches/100_add_print_file_row_table.sql:4]: we already have a patch 100 merged into `main`

File: patches/100_add_print_file_row_table.sql [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+create table casev3.print_file_row
+(
+    id             bigserial    not null,
+    batch_id       uuid,
+    batch_quantity int4         not null,
+    pack_code      varchar(255) not null,
+    print_supplier varchar(255) not null,
+    row            varchar(255),
+    action_rule_id uuid,
```
Comment (NickGrantONS) [inline patches/100_add_print_file_row_table.sql:17]: remove the action rule - not needed

File: patches/100_add_print_file_row_table.sql [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    primary key (id)
+);
+
+alter table if exists casev3.print_file_row
+    add constraint FKfo2cs8srx9nsfeliw3ekveehu
+        foreign key (action_rule_id)
+            references casev3.action_rule;
```
Comment (NickGrantONS) [inline patches/100_add_print_file_row_table.sql:24]: remove the action rule - not needed

File: groundzero_ddl/ddl_version.sql [hunk 6]
```diff
@@ -3,5 +3,5 @@ CREATE TABLE ddl_version.version (version_tag varchar(256) PRIMARY KEY, updated_
 
 -- Version and patch number for the current ground zero,
 -- NOTE: These must be updated every time the repo is tagged
-INSERT INTO ddl_version.patches (patch_number, applied_timestamp) VALUES (0, current_timestamp);
-INSERT INTO ddl_version.version (version_tag, updated_timestamp) VALUES ('v1.0.0', current_timestamp);
+INSERT INTO ddl_version.patches (patch_number, applied_timestamp) VALUES (100, current_timestamp);
```
Comment (NickGrantONS) [inline groundzero_ddl/ddl_version.sql:6]: we already have a patch 100

**Review by NickGrantONS — CHANGES_REQUESTED** [review]
See comments

**Review by NickGrantONS — APPROVED** [review]

**Review by ryangrundy7 — APPROVED** [review]

**Pull request merged**